adapter/src/test/java/com/twilio/kudu/sql/SortedAggregationIT.java

```diff
@@ -345,7 +400,7 @@ public void aggregateSortedResultsByAccountWrongDirection() throws Exception {
       assertTrue("Should have results to iterate over", queryResult.next());
       assertFalse(String.format("Plan should not contain KuduSortRel. It is\n%s", plan), plan.contains("KuduSortRel"));
       assertEquals("Full SQL plan has changed\n", expectedPlan, plan);
-      assertEquals("Incorrect aggregated value", 3662, queryResult.getLong(2));
+      assertEquals("Incorrect aggregated value", 5602, queryResult.getLong(2));
```

Comment:
These magic values changing has me tickled. Did we introduce a regression with this change? Did we calculate it wrong in prior versions?

Reply:
I changed the data in the table because of the new test that I added, so I had to update some of the existing tests from 
```
      insertRow(stmt, 5, 31, 101, 30L, "message-body", ACCOUNT_SID1);
      insertRow(stmt, 5, 31, 102, 30L, "message-body", ACCOUNT_SID1);
```
to
```
      insertRow(stmt, 5, 31, 101, 30L, "user-login", ACCOUNT_SID1);
      insertRow(stmt, 5, 31, 102, 30L, "user-login", ACCOUNT_SID1);
```